components/automate-gateway/eventfeed/event_guitar_strings.go

```diff
@@ -2,22 +2,15 @@ package eventfeed
 
 import (
 	"context"
-	"fmt"
```

Comment:
All the gateway code is simpler now because we are only pulling events from one service.